crates/nu-command/Cargo.toml

```diff
@@ -88,6 +88,7 @@ titlecase = "2.0.0"
 toml = "0.5.8"
 unicode-segmentation = "1.8.0"
 url = "2.2.1"
+url-escape = "0.1.1"
```

Comment:
Is there a reason for choosing the [`url-escape`](https://lib.rs/crates/url-escape) crate?

It's not widely used (~2000 downloads/month); did you consider [`percent-encoding`](https://lib.rs/crates/percent-encoding) or [`urlencoding`](https://lib.rs/crates/urlencoding)
 instead?

Reply:
Ah, yes you are right. will change it to use [`urlencoding`](https://lib.rs/crates/urlencoding)

Reply:
Mhh so if I grok it correctly from the docs the `url-escape` crate supports maintaining the path components while the others encode full urls to be used as a component/query?

Reply:
yep, that's true @sholderbach . But @rgwood does make a good point about it not being widely used. Instead with any of the other 2 libraries mentioned, we can customize the code anyway as I did with latest code in this PR.